doc/sphinx-guides/source/api/changelog.rst

```diff
@@ -7,12 +7,16 @@ This API changelog is experimental and we would love feedback on its usefulness.
     :local:
     :depth: 1
 
+v6.5
+----
+
+- **/api/datasets/{identifier}/links**: The GET endpoint returns a list of Dataverses linked to the given Dataset. The format of the response has changes for v6.4 making it backward incompatible.
```

Comment:
Should this say "has changes for v6.5" ?

Reply:
Yes! Thanks! Fixed in 38cff71.